library/packages/src/modules/PackageSystem.rb

```diff
@@ -66,7 +67,7 @@ def main
       Yast.include self, "packages/common.rb"
 
       @_rpm_query_binary_initialized = false
-      @_rpm_query_binary = "rpm"
+      @_rpm_query_binary = "/usr/bin/rpm"
```

Comment:
Again, this is beyond the scope of this PR, but still...

It seems to be very inconsistent to have a member variable (why not a constant? does that ever change?) for the RPM command, yet not use that variable, but the string literal everywhere in the code. That makes me wonder what that member variable is good for at all.

Reply:
here it is problematic, as in code this path can change. 
see:
https://github.com/yast/yast-yast2/pull/872/files/7f355a06aa1746e0c11fb8315ca9bf2bd655f296#diff-2a6112a932a3433114e3b20bb80f0c42R323
but it looks quite old feature. So for now I am not sure if I want to drop that feature completely.

Reply:
OK. In case you decide to keep `InitRPMQueryBinary`, please fix `/bin` to `/usr/bin` here, twice: https://github.com/yast/yast-yast2/blame/7f355a06aa1746e0c11fb8315ca9bf2bd655f296/library/packages/src/modules/PackageSystem.rb#L330